app/src/pages/Play/GameRoundInstructionCard.tsx

```diff
@@ -43,7 +49,10 @@ function GameRoundInstructionCard(props: {
           {props.round}
         </Typography>
       </Box>
-      <Box p={1}>{GameRoundDescription[props.round]}</Box>
+      <Box p={1}>
+        {GameRoundDescription[props.round as GameRound] ||
```

Comment:
`GameRoundDescription[props.round as GameRound]` will return undefined if props.round is not actually a GameRound, and some other string.